configs/ray-data-batch-classification/aws.yaml

```diff
@@ -0,0 +1,8 @@
+head_node_type:
+  name: head
+  instance_type: g5.xlarge
```

Comment:
Can this use the serverless auto select workers mode? We are trying to push away from GPU head nodes due to idle cost, especially when using these for community edition type use cases.

Reply:
Yes, definitely.

Reply:
is this serverless now? seems that it is still using gpu head node.

Reply:
I think what @shomilj meant is to not use a gpu node as head node, and run workloads on worker nodes with auto gpu node selection.